cadence/contracts/bridge/FlowEVMBridgeUtils.cdc

```diff
@@ -424,6 +425,27 @@ contract FlowEVMBridgeUtils {
         )
     }
 
+    /// Retrieves the EVMPointer view from a given type's defining contract if the view is supported.
+    /// NOTE: This does not guarantee the association is valid, only that the defining Cadence contract declares
+    /// the association.
+    ///
+    /// @param from: The type for which to retrieve the EVMPointer view
+    ///
+    /// @return The resolved EVMPointer view for the given type or nil if the view is unsupported
+    ///
+    access(all)
+    fun getEVMPointer(forType: Type): CrossVMMetadataViews.EVMPointer? {
```

Comment:
Rename to `getEVMPointerView`

Reply:
Updated in [3d61754](https://github.com/onflow/flow-evm-bridge/pull/173/commits/3d6175476c78a4dcc6733374553498469405bc68)